posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -17,33 +17,26 @@ Reece Nana <https://github.com/reece-nana>
-** Jakarta Interceptors 2.1: picking up the changes from Annotation 2.1
-
-In addition to the Core Profile features, Jakarta EE 10 Expression Language 5.0 is included in this beta and provides new features and specification clarifications.  Likewise, Jakarta Concurrency 3.0 is also included and enhances the set of parallel and reactive programming models available to applications, enabling tasks and completion stage actions to run with the thread context of the submitter.
-
-But it's not only Jakarta EE features that are debuting in this release!  MicroProfile OpenAPI is updated to version 3.1, in which you can specify optional authentication for operations and multiple annotation enhancements.  Last but not least, Password Utilities 1.1 is introduced in this beta, which no longer forcefully federates stand-alone user registries.
+As well as Jakarta EE 10 Core Profile, we've included in this beta two other Jakarta EE features: Jakarta EE 10 Expression Language 5.0, which means you can access any annotations present on a particular method using the `getAnnotations()` method, and more; and Jakarta Concurrency 3.0, which enhances the set of parallel and reactive programming models available to applications, and more.
 
+But it’s not only Jakarta EE features in this release: MicroProfile OpenAPI 3.1 provides the ability to specify optional authentication for operations and multiple annotation enhancements, and Password Utilities 1.1 no longer forcefully federates stand-alone user registries.
 
 The link:{url-about}[Open Liberty] 22.0.0.9-beta includes the following beta features (along with link:{url-prefix}/docs/latest/reference/feature/feature-overview.html[all GA features]):
 
-* Jakarta EE 10 Core Profile
-** <<restful, RESTful Web Services 3.1>>
-** <<json, Jakarta JSON Binding and Processing>>
-** <<cdi, CDI 4.0>>
-* <<expression, Jakarta EE 10 Expression Language 5.0>>
-* <<concurrency, Jakarta EE 10 Concurrency 3.0>>
-* <<microprofile, MicroProfile OpenAPI 3.1>>
-* <<password, Password Utilities 1.1>>
+* Jakarta EE 10 Core Profile:
+** <<restful, Support for multipart/form-data content type (Jakarta RESTful Web Services 3.1)>>
+** <<json, Enhanced JSON string parsing and new serialization/deserialization options (Jakarta JSON Processing 2.1 and Jakarta JSON Binding 3.0)>>
+** <<cdi, Support for build compatible extensions and observable container state events (Jakarta CDI 4.0)>>
+* Jakarta EE 10:
+** <<expression, Access any available annotations using getAnnotations() (Jakarta Expression Language 5.0)>>
+** <<concurrency, Enhanced parallel and reactive programming models available to applications (Jakarta Concurrency 3.0)>>
+* <<microprofile, Document APIs where authentication is optional or complex (MicroProfile OpenAPI 3.1)>>
+* <<password, Stand-alone user registries no longer forcefully federated (Password Utilities 1.1)>>
```

Comment:
I know in general we want to lead with functionality and not the details of what feature is responsible for providing them.  However, I wonder if this is an exception where the features are the main story because the headline is 'Jakarta EE Core Profile' and these are the features that make it up.  I would suggest considering doing something like 
 `** <<restful, Jakarta RESTful Web Services 3.1: Support for multipart/form-data content type>>`

Reply:
I kind-of agree but I think it still works this way. While only existing Jakarta EE users will probably be interested in a beta of this, it'd be nice to not continue obfuscating what Jakarta EE actually gives to you - there'll be features that even existing users aren't that familiar with. Also, it'd be good to not make style exceptions (unless it's really incorrect) for the person compiling the release posts to have to deal with.